src/App.js

```diff
@@ -1,23 +1,46 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState } from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Search from './components/Search';
+
 
 function App() {
+  const [dishes, setDishes] = useState(foods)
+  const createFood = (newDishes)=>{
+    const edit = [newDishes, ...dishes]
+    setDishes(edit)
+  }
+const searchFood = (searchQuery)=>{
+  const filteredFood = dishes.filter((food) => 
+  food.name.toLowerCase().includes(searchQuery.toLowerCase())
+  );
+  setDishes(filteredFood)
+}
+const deleteFood = (name)=>{
+  const filteredFood = dishes.filter((dishes) => dishes.name !== name);
+  setDishes(filteredFood)
+}
+
+
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="container">
+        <Search filterFood={searchFood}/>
+        <AddFoodForm createFood= {createFood}/>
+      <div className="App">
+
+       { dishes.map((dish) => {
```

Comment:
Last but not least, for your search bar to work, you now need to map over the `showFoods` instead of the `dishes`:
`{ showFoods.map((dish) => {`